usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -60,15 +72,8 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
         )
     )
 
-    scap_report = os.environ.get("SCAP_REPORT", "")
-    if scap_report:
-        if os.path.exists(scap_report) or not scap_report.endswith(".html"):
-            print("Ignoring SCAP_REPORT: {}".format(scap_report))
-        else:
-            scap_report = "--report {}".format(scap_report)
-
     result = host.run(
-        "sudo oscap xccdf eval {scap_report} --local-files {dir} --profile {profile} {file}".format(
+        "sudo oscap xccdf eval --report {scap_report} --local-files {dir} --profile {profile} {file}".format(
```

Comment:
If environment variable isn't set this will format to `--report --local-files` leading to an error?

Reply:
True. Fixed.